### PR TITLE
fix: use @einride/prettier-config when formatting markdown

### PR DESCRIPTION
**Pull request by filiptammergard**
Solves https://github.com/einride/mage-tools/pull/41#discussion_r775265477

### CODE REVIEW

File: targets/mgprettier/targets.go [hunk 1]
```diff
@@ -14,17 +14,22 @@ import (
...
+
 var executable string
 
 func FormatMarkdown(ctx context.Context) error {
 	logger := mglog.Logger("prettier")
 	ctx = logr.NewContext(ctx, logger)
-	mg.CtxDeps(ctx, prepare)
+	prettierrc := filepath.Join(mgtool.GetPath(), "prettier", ".prettierrc.js")
+	mg.CtxDeps(ctx, mg.F(prepare, prettierrc))
 	args := []string{
```
Comment (odsod) [inline targets/mgprettier/targets.go:33]: @filiptammergard I think all that's needed to get it working is to add a flag here that points to the newly written `prettierrc` path

Comment (filiptammergard) [reply targets/mgprettier/targets.go:33]: That's right! Added it and now it formats markdown correctly 👍

Comment (odsod) [reply targets/mgprettier/targets.go:33]: Awesome!!

**Review by odsod — APPROVED** [review]
Great stuff!

**Review by Edholm — APPROVED** [review]
Nice!

**Comment by viktorvoltaire** [comment]
Awesome @filiptammergard 👌

**Pull request merged**